reframe/utility/__init__.py

```diff
@@ -48,7 +48,7 @@ def import_module_from_file(filename):
         filename = os.path.abspath(filename)
 
     module_name = _get_module_name(filename)
-    if os.path.isabs(filename):
+    if os.path.isabs(filename) or os.path.basename(filename) == filename:
```

Comment:
This fix does not work. Try running reframe from a directory up:

```
./reframe/bin/reframe --system=generic -c reframe/tutorial/example1.py -l
```

The whole problem comes from how the `sys.path` is defined (check [this](https://docs.python.org/3.6/library/sys.html#sys.path)). Actually, for our case, the first element is the path where the `reframe.py` or `test_reframe.py` exists. That's why these relative imports work fine when we we invoke ReFrame from the same directory. I've done a PR into your branch with the correct implementation.